Philips.CodeAnalysis.MaintainabilityAnalyzers/Maintainability/AvoidProblematicUsingPatternsAnalyzer.cs

```diff
@@ -72,13 +74,32 @@ private bool IsProblematicExpression(ExpressionSyntax expression)
 					return true;
 				}
 
-				if (symbolInfo.Symbol is ILocalSymbol)
+				if (symbolInfo.Symbol is ILocalSymbol localSymbol)
 				{
+					// Check if this local symbol is declared as an out parameter
+					if (IsOutParameterDeclaration(localSymbol))
+					{
+						return false; // Out parameters are safe to use
+					}
 					return true;
 				}
 			}
 
 			return false;
 		}
+
+		private bool IsOutParameterDeclaration(ILocalSymbol localSymbol)
```

Comment:
Is there a natural home for this in one of our helper classes, to facilitate reuse?